src/helpers/images.mjs

```diff
@@ -0,0 +1,63 @@
+import { readdir, rename } from "fs/promises";
+import { dirname, join } from "path";
+
+const imageExtensions = /\.(jpg|jpeg)$/i;
+const imageBaseName = /^image[1-9]{1}[0-9]*\.jpg$/;
+
+/**
+ * @description Reads every file in a folder that in the format "image{number}.jpg"
+ * and returns them as an array of absolute file names.
+ * @param {string} dirPath - The absolute path of the input folder.
+ * @returns {string[]} - An array of all absolute image filenames
+ * that are in the format "image{number}.jpg".
+ */
+export async function readImagesFromDir(dirPath) {
+  const allFiles = await readdir(dirPath);
+
+  const imageFiles = allFiles
+    .filter((file) => {
+      return imageBaseName.test(file);
+    })
+    .sort((a, b) => {
+      const [numberA, numberB] = [
+        parseInt(a.match(/\d+/)[0]),
+        parseInt(b.match(/\d+/)[0]),
+      ];
+      return numberA - numberB;
+    });
+
+  return imageFiles.map((image) => join(dirPath, image));
+}
+
+/**
+ * @description A function to rename any .jpg/.jpeg files in a folder to the format "image{number}.jpg".
+ * @param {string} dirPath The absolute path of the input folder.
+ * @returns {number} The number of renamed images in the folder.
+ */
+export async function renameImages(dirPath) {
+  const allFiles = await readdir(dirPath);
+  let startCount = allFiles
+    .filter((file) => imageBaseName.test(file))
+    .reduce((max, file) => {
+      const number = parseInt(file.match(/\d+/)[0]);
+      return number > max ? number : max;
+    }, 0);
+
+  const imageFiles = allFiles.reduce((res, file) => {
+    if (
+      imageExtensions.test(file) &&
```

Comment:
please lower case `file` before testing the regex.
That is because sometimes, the image extension looks like this |JPG|JPEG...
or even like this jPG|Jpeg|pNg ...

Reply:
oh my bad, forgot to add case insensitive to
```js 
const imageBaseName = /^image[1-9]{1}[0-9]*\.jpg$/;
```
will do it now in a next commit.

Reply:
Done, should be working well now if I'm not missing anything.